src/main/java/utils/ApiHelper.java

```diff
@@ -0,0 +1,95 @@
+package utils;
+
+import api.ProductCode;
+import api.ProductQuantity;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import io.restassured.module.jsv.JsonSchemaValidator;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiHelper {
+    private RequestSpecification requestSpecification;
+    public Map<String, Object> storage = new HashMap<>();
+
+    public void createCart(String url) {
+        requestSpecification = RestAssured.given()
```

Comment:
setup of RequestSpecification should be somewhere in test hooks, since it's not related only to cart related classes, but for all the project resources